ktor-io/posix/src/io/ktor/utils/io/locks/Synchronized.kt

```diff
@@ -217,18 +217,13 @@ public actual open class SynchronizedObject {
         }
     }
 
-    @OptIn(FreezingIsDeprecated::class)
     protected class LockState(
         public val status: Status,
         public val nestedLocks: Int,
         public val waiters: Int,
         public val ownerThreadId: pthread_t? = null,
         public val mutex: CPointer<ktor_mutex_node_t>? = null
-    ) {
-        init {
-            freeze()
```

Comment:
`freeze()` is deprecated with deprecation level ERROR, and the deprecation message says it could be safely removed.